sites/page-details/index.json-ld

```diff
@@ -82,5 +83,346 @@
 				"path": "deprecated/page-details-v1-fr.html"
 			}
 		]
-	}
+	},
+	"dependencies": [
+		"Page feedback area",
+		"Share this page",
+		"Date modified"
+	],
+	"a11yGuidance": "no accessibility guidance",
+	"variations": [
+		{
+			"name": {
+				"en": "Page details with Page feedback tool",
+				"fr": "Détails de la page avec Outil de rétroaction sur la page"
```

Comment:
Not exactly accurate,

This variations seem to include 4 inner variation. It will be preferable to move those inner variation as normal variation.